src/invidious/views/search.ecr

```diff
@@ -1,4 +1,10 @@
+<%
+ # Satify compiler  
+%>
+<% filter_params = nil %>
```

Comment:
you can use a single `<% %>` pair for the comment and the variable. This generates less empty lines in the output.